### DISCUSSION TITLE
Zugriff Benutzerbestätigung

### DISCUSSION THREAD

**Discussion by klar574** [immense/Remotely#886] [Q&A]
Hallo zusammen,

ich habe Remotely auf meiner NAS installiert. Läuft soweit alles gut. Ich hätte nur gerne das der Zugriff nicht ohne eine Benutzerbestätigung erfolgen kann. Jetzt komme ich beim Verbinden direkt immer auf das Gerät. Gibt es hier eine Möglichkeit in dem Install Skript etwas anzupassen, damit der Benutzer den Zugriff zu seinem Gerät bestätigen muss!? Oder kann ich das noch irgendwo einstellen? Gruß Jens 

Hello everyone,

 I installed Remotely on my NAS.  Everything is going well so far.  I would just like access not to be possible without user confirmation.  Now I always go directly to the device when connecting.  Is there a way to adjust something in the install script so that the user has to confirm access to their device!?  Or can I set that somewhere else?  Greetings, Jens

**Comment by gravasio** [comment]
please write in English!

**Comment by gravasio** [comment]
Hi Jens, 
you can achieve what you're asking enabling:
"Server Config"->"Enforce Attended Access"
Unfortunately this can be set only server wide and cannot be enabled or disabled for specific groups.

**Comment by klar574** [comment]
Ok, unfortunately that's not ideal either. The desktop agent is also installed. But it asks for the server address when starting. Can the address be included in the script so that the user doesn't have to enter it?

**Reply by gravasio** [reply]
Hello Klar, that’s strange and shouldn’t be happening. The server address should be included in the installation script. If the desktop agent is asking for the server address at startup, there might have been an issue during the installation.
Did you install the agent using the Powershell script found in the download section? If not, I would suggest trying to reinstall the agent using that script

**Comment by klar574** [comment]
Hello gravasio, I have now deleted everything on a device and reinstalled it using the Powershell script. The device appears in Remotely and I can connect without asking. When I start Remotely Desktop, the application then starts with the query for the server URL
